groups/xml-infrastructure/templates/heritage-live-eu-west-2/bep_cron.tpl

```diff
@@ -65,7 +65,7 @@
 4,34 * * * * /home/xml/supportscripts/filinqueuecheck.sh email 100 >/dev/null 2>&1
 0 8-17 * * * /home/xml/supportscripts/updateXMLFormDetail.sh >/dev/null 2>&1
 
-30 00 * * * /home/xml/efbackend/ef_presenter_data_import.sh ${ EF_PRESENTER_DATA_BUCKET } live-presenter-data-\$(date --date="-1 day" '+\%Y-\%m-\%d') >/dev/null 2>&1
+30 00 * * * /home/xml/efbackend/ef_presenter_data_import.sh ${ EF_PRESENTER_DATA_BUCKET } live-presenter-data-$(date --date="-1 day" '+\%Y-\%m-\%d') >/dev/null 2>&1
```

Comment:
We may also need to remove the escaping in the `date` command

Reply:
We'll need to retain the escape characters in the date format string since percent symbols have a special meaning in crontab files. `crontab(5)` says:

> Percent-signs (%) in the command, unless escaped with backslash (\), will be changed into newline  characters,  and  all data after the first % will be sent to the command as standard input.

The changes have been tested on one of the `xml-bep` systems and work as expected.